lib/double_entry/reporting/aggregate_array.rb

```diff
@@ -50,8 +53,9 @@ def retrieve_aggregates
         scope = LineAggregate.
                 where(:function => function).
                 where(:range_type => 'normal').
-                where(:account => account.to_s).
-                where(:code => code.to_s).
+                where(:account => account.try(:to_s)).
+                where(:code => code.try(:to_s)).
+                where(:partner_account => partner_account.try(:to_s)).
```

Comment:
Going off of what was here before, will this change behavior slightly since it'll fetch line aggregates without a partner account when `nil` as opposed to fetching all of them?